.github/workflows/package-build.yaml

```diff
@@ -30,9 +30,10 @@ jobs:
 
       - name: Set up Python 3.x
         uses: actions/setup-python@v5
+        id: setup-python
```

Comment:
Probably need this ID in the other setup python step(s) as well. 
https://github.com/oqc-community/qat/pull/163/files#diff-39100e6539ba9a86c5830cf257ebdfdef8797d743f9787ec1be460f1e8dc941aR79